dist/src/main/java/io/camunda/zeebe/gateway/TopologyServices.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.gateway;
+
+import io.atomix.cluster.AtomixCluster;
+import io.atomix.cluster.ClusterMembershipService;
+import io.atomix.cluster.messaging.ClusterCommunicationService;
+import io.camunda.zeebe.gateway.impl.broker.cluster.BrokerTopologyManager;
+import io.camunda.zeebe.gateway.impl.broker.cluster.BrokerTopologyManagerImpl;
+import io.camunda.zeebe.scheduler.ActorScheduler;
+import io.camunda.zeebe.topology.GatewayClusterTopologyService;
+import io.camunda.zeebe.topology.gossip.ClusterTopologyGossiperConfig;
+import java.time.Duration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TopologyServices {
+  private final ActorScheduler scheduler;
+  private final ClusterMembershipService clusterMembershipService;
+  private final ClusterCommunicationService clusterCommunicationService;
+
+  @Autowired
+  public TopologyServices(final ActorScheduler scheduler, final AtomixCluster atomixCluster) {
+    this.scheduler = scheduler;
+    clusterMembershipService = atomixCluster.getMembershipService();
+    clusterCommunicationService = atomixCluster.getCommunicationService();
+  }
+
+  @Bean
+  GatewayClusterTopologyService gatewayClusterTopologyService() {
```

Comment:
Didn't this start it also on the broker? If yes, then there will still be two gossiper running in broker, Right? One from this `GatewayClusterTopologyService` and one from the `ClusterTopologyManagerService`.

Reply:
Got it. There is only started for standalone gateway because of spring only scan this package when starting StandaloneGateway. :+1: